source/lib/predicates/number.ts

```diff
@@ -17,7 +17,7 @@ export class NumberPredicate extends Predicate<number> {
 	 */
 	inRange(start: number, end: number) {
 		return this.addValidator({
-			message: value => `Expected ${value} to be in range [${start}..${end}]`,
+			message: (value, label) => `Expected ${label} ${value} to be in range [${start}..${end}]`,
```

Comment:
Same as with the boolean type. Should we return the following message instead?

> Expected ${label} to be in range [${start}..${end}], got ${value}

Result
> Expected foo to be in range [0..9], got 10
> Expected number to be in range [0..9], got 10

Instead of
> Expected foo 10 to be in range [0..9]
> Expected number 10 to be in range [0..9]

-----
If we do it for this one, we should do it for all the rest in this predicate as well.

> Expected foo to be greater than 9, got 10
...

Reply:
yeah, i think this wording is better -- updated :)